config.yml

```diff
@@ -1,9 +1,10 @@
 ---
 defaults:
-  # TaskCluster purge cache configuration
-  purgeCache:
-    # Prefix for exchanges declared
+  app:
     exchangePrefix:               'v1/'
+    cacheBusterTableName:         'CacheBusters'
```

Comment:
How about `cachePurgeTableName`, hmm... just trying to help with naming, hehe :)

(it couldn't be less important)